<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Atenea</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/icons/icons.qrc">
    <normaloff>:/icons/pantheon.png</normaloff>:/icons/pantheon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="atheneaStages">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="MainPage">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QTabWidget" name="tabWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="buildTab">
           <attribute name="title">
            <string>Export</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4" stretch="5,0,0,2,0">
            <item>
             <layout class="QHBoxLayout" name="buildInfoLayout" stretch="3,0,7,0,0">
              <item>
               <layout class="QVBoxLayout" name="buildFiltersLayout" stretch="0,10">
                <item>
                 <layout class="QHBoxLayout" name="filterByText">
                  <item>
                   <widget class="QLabel" name="filterTitle">
                    <property name="enabled">
                     <bool>true</bool>
                    </property>
                    <property name="text">
                     <string>Search:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="buildFilterSearch">
                    <property name="enabled">
                     <bool>true</bool>
                    </property>
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>10</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QScrollArea" name="filtersScrollArea">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="widgetResizable">
                   <bool>true</bool>
                  </property>
                  <widget class="QWidget" name="filtersScrollAreaLayout">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>193</width>
                     <height>378</height>
                    </rect>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_2">
                    <item>
                     <widget class="QTreeWidget" name="filtersTree">
                      <property name="enabled">
                       <bool>false</bool>
                      </property>
                      <property name="uniformRowHeights">
                       <bool>true</bool>
                      </property>
                      <property name="animated">
                       <bool>true</bool>
                      </property>
                      <property name="columnCount">
                       <number>1</number>
                      </property>
                      <column>
                       <property name="text">
                        <string>Filters</string>
                       </property>
                      </column>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="Line" name="Separator_1">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="buildNodesTreeLayout">
                <item>
                 <widget class="QStackedWidget" name="buildNodesStackedWidget">
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="currentIndex">
                   <number>0</number>
                  </property>
                  <widget class="QWidget" name="buildTreeModelPage">
                   <layout class="QVBoxLayout" name="verticalLayout_5">
                    <item>
                     <widget class="QTreeWidget" name="buildNodesTree">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="selectionBehavior">
                       <enum>QAbstractItemView::SelectItems</enum>
                      </property>
                      <property name="uniformRowHeights">
                       <bool>false</bool>
                      </property>
                      <property name="animated">
                       <bool>true</bool>
                      </property>
                      <property name="headerHidden">
                       <bool>false</bool>
                      </property>
                      <attribute name="headerDefaultSectionSize">
                       <number>375</number>
                      </attribute>
                      <column>
                       <property name="text">
                        <string>Art</string>
                       </property>
                      </column>
                      <column>
                       <property name="text">
                        <string>Status</string>
                       </property>
                      </column>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                  <widget class="QWidget" name="buildFilteredListPage">
                   <layout class="QVBoxLayout" name="verticalLayout_6">
                    <item>
                     <widget class="QListWidget" name="buildFilteredItemList">
                      <property name="showDropIndicator" stdset="0">
                       <bool>false</bool>
                      </property>
                      <property name="selectionMode">
                       <enum>QAbstractItemView::SingleSelection</enum>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="Line" name="Separator_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="ItemsToBuild">
                <item>
                 <widget class="QLabel" name="itemsToBuildTitle">
                  <property name="text">
                   <string>Items To Build:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QListWidget" name="itemsList">
                  <property name="editTriggers">
                   <set>QAbstractItemView::NoEditTriggers</set>
                  </property>
                  <property name="showDropIndicator" stdset="0">
                   <bool>false</bool>
                  </property>
                  <property name="defaultDropAction">
                   <enum>Qt::IgnoreAction</enum>
                  </property>
                  <property name="alternatingRowColors">
                   <bool>true</bool>
                  </property>
                  <property name="selectionMode">
                   <enum>QAbstractItemView::SingleSelection</enum>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>16</width>
                    <height>16</height>
                   </size>
                  </property>
                  <property name="verticalScrollMode">
                   <enum>QAbstractItemView::ScrollPerPixel</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="buildFilteredItempath">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="buildButtonsLayout" stretch="1,2,2">
              <item>
               <widget class="QPushButton" name="buttonBuildRefresh">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>1</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Refresh</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="buttonBuildStep1">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>1</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Build Step 1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="buttonBuildStep2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>1</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Build Step 2</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="buildConsoleLayout" stretch="0">
              <item>
               <widget class="QScrollArea" name="scrollArea">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>1</horstretch>
                  <verstretch>1</verstretch>
                 </sizepolicy>
                </property>
                <property name="widgetResizable">
                 <bool>true</bool>
                </property>
                <widget class="QWidget" name="scrollAreaWidgetContents">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>943</width>
                   <height>161</height>
                  </rect>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <widget class="QLabel" name="consoleOutput">
                    <property name="text">
                     <string/>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
                    </property>
                    <property name="wordWrap">
                     <bool>true</bool>
                    </property>
                    <property name="openExternalLinks">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QProgressBar" name="buildProgressBar">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="inputMethodHints">
               <set>Qt::ImhNone</set>
              </property>
              <property name="value">
               <number>0</number>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="textVisible">
               <bool>true</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="invertedAppearance">
               <bool>false</bool>
              </property>
              <property name="textDirection">
               <enum>QProgressBar::TopToBottom</enum>
              </property>
              <property name="format">
               <string>%p%</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="settingsTab">
           <attribute name="title">
            <string>Settings</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QVBoxLayout" name="settingsLayout">
              <property name="sizeConstraint">
               <enum>QLayout::SetMaximumSize</enum>
              </property>
              <item>
               <widget class="QFrame" name="pathsContainer">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_9">
                 <property name="sizeConstraint">
                  <enum>QLayout::SetMinimumSize</enum>
                 </property>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="VerticalSpace">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="buttonsLayout">
              <property name="sizeConstraint">
               <enum>QLayout::SetMinimumSize</enum>
              </property>
              <item>
               <widget class="QPushButton" name="buttonApply">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Apply</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="buttonCancel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Cancel</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="moreToComeTab">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <attribute name="title">
            <string>More To Come</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="WIPLabel">
              <property name="text">
               <string>NEW TOOLS COMMING!</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="projectTitle">
          <property name="font">
           <font>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
            <underline>true</underline>
           </font>
          </property>
          <property name="text">
           <string>Project Name</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>tabWidget</zorder>
       <zorder>tabWidget</zorder>
       <zorder>projectTitle</zorder>
      </widget>
      <widget class="QWidget" name="WelcomePage">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QWidget" name="Login" native="true">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>119</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <pointsize>59</pointsize>
              </font>
             </property>
             <property name="text">
              <string>ATHENEA</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="LogiBox" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QWidget" name="userWidget" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>User</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="userLine">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>300</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>AtheneaUser</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="Password" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>Password</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="passwordLine">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>300</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>AtheneaUser+</string>
                   </property>
                   <property name="echoMode">
                    <enum>QLineEdit::Password</enum>
                   </property>
                  </widget>
                 </item>
                </layout>
                <zorder>label_2</zorder>
                <zorder>projects</zorder>
                <zorder>passwordLine</zorder>
               </widget>
              </item>
              <item>
               <widget class="QScrollArea" name="projects">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="widgetResizable">
                 <bool>true</bool>
                </property>
                <widget class="QWidget" name="scrollAreaWidgetContents_2">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>933</width>
                   <height>210</height>
                  </rect>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_10">
                  <item>
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>Projects</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QListWidget" name="projectList"/>
                  </item>
                 </layout>
                 <zorder>Login</zorder>
                 <zorder>projectList</zorder>
                 <zorder>label_5</zorder>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="loginButton">
                <property name="text">
                 <string>Login</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>242</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1007</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Pantheon"/>
    <addaction name="separator"/>
    <addaction name="actionReport_an_Issue"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout_Pantheon">
   <property name="text">
    <string>About Pantheon</string>
   </property>
  </action>
  <action name="actionReport_an_Issue">
   <property name="text">
    <string>Report an Issue</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources/icons/icons.qrc"/>
  <include location="../../resources/fileStatuses/fileStatuses.qrc"/>
  <include location="../../resources/statuses/statuses.qrc"/>
 </resources>
 <connections/>
</ui>
